kratos/processes/simple_mortar_mapper_process.cpp

```diff
@@ -89,6 +118,25 @@ void SimpleMortarMapperProcess<TDim, TNumNodes, TVarType, TNumNodesMaster>:: Exe
     else
         ExecuteImplicitMapping();
 
+    // We apply the coeffcient if different of one
+    if (mMappingCoefficient != 1.0) {
+        NodesArrayType& r_nodes_array = mDestinationModelPart.Nodes();
+
+        if (mDestinationHistorical) {
+            #pragma omp parallel for
+            for (int k = 0; k< static_cast<int> (r_nodes_array.size()); ++k) {
+                auto it_node = r_nodes_array.begin() + k;
+                it_node->FastGetSolutionStepValue(mDestinationVariable) *= mMappingCoefficient;
+            }
+        } else {
+            #pragma omp parallel for
+            for (int k = 0; k< static_cast<int> (r_nodes_array.size()); ++k) {
+                auto it_node = r_nodes_array.begin() + k;
+                it_node->GetValue(mDestinationVariable) *= mMappingCoefficient;
```

Comment:
are u sure that mDestinationVariable exists?

Reply:
Yes, it is set previously